extension/src/background/helpers/dataStorage.ts

```diff
@@ -252,12 +252,17 @@ export const addIsNonSSLEnabled = async () => {
   }
 };
 
-export const addIsBlockaidAnnounced = async () => {
+export const removeStellarExpertData = async () => {
```

Comment:
since we're going to launch Blockaid silently - I'm removing the migration needed to show the modal. I'm lumping it into this PR because I wanted to add a cleanup migration and I wanted to use the migration version number I had previously added for the Blockaid announcement